compare_mt/compare_mt_main.py

```diff
@@ -429,6 +470,12 @@ def main():
                       help="Number of decimals to print for floating point numbers")
   parser.add_argument('--scorer_scale', type=float, default=100, choices=[1, 100],
                       help="Set the scale of BLEU, METEOR, WER and chrF to 0-1 or 0-100 (default 0-100)")
+  parser.add_argument('--lang_id', type=str, nargs='*',
+                      default=['model=model,min_length=min_length,print_lines=False,print_line_numbers=False'],
```

Comment:
Do I understand correctly that the default is to include the lang id report? How do I turn it off?

Reply:
done